sdk/src/test/java/io/dapr/it/DaprIntegrationTestingRunner.java

```diff
@@ -0,0 +1,155 @@
+package io.dapr.it;
+
+import org.junit.Assert;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.*;
+
+
+public class DaprIntegrationTestingRunner {
+
+    public static   DaprIntegrationTestingRunner.DaprFreePorts DAPR_FREEPORTS;
+
+    static {
+        try {
+            DAPR_FREEPORTS = new DaprIntegrationTestingRunner.DaprFreePorts().initPorts();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    private Runtime rt = Runtime.getRuntime();
+    private Process proc;
+
+    private String successMessage;
+    private Class serviceClass;
+    private Boolean useAppPort;
+    private int sleepTime;
+    private String appName;
+
+    DaprIntegrationTestingRunner(String successMessage, Class serviceClass, Boolean useAppPort, int sleepTime) {
+        this.successMessage = successMessage;
+        this.serviceClass = serviceClass;
+        this.useAppPort = useAppPort;
+        this.sleepTime = sleepTime;
+        this.generateAppName();
+    }
+
+
+
+    public DaprFreePorts initializeDapr() throws Exception {
+
+
+        String daprCommand=this.buildDaprCommand();
+        System.out.println(daprCommand);
+        proc= rt.exec(daprCommand);
```

Comment:
Use ProcessBuilder instead. This way we don't need to forcefully set an env variable anymore.
https://docs.oracle.com/javase/7/docs/api/java/lang/ProcessBuilder.html